Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1332,7 +1332,10 @@ extension BrowserViewController: URLBarDelegate {
 
         updateFindInPageVisibility(visible: false)
         
-        let shouldShowChronTabs = UIDevice.current.userInterfaceIdiom != .pad && profile.prefs.boolForKey(PrefsKeys.ChronTabsPrefKey) ?? (chronTabsUserResearch?.chronTabsState ?? false)
+        let shouldShowChronTabs = AppConstants.CHRONOLOGICAL_TABS // default on in Beta
+                                  && UIDevice.current.userInterfaceIdiom != .pad // only iPhone
```

Comment:
```suggestion
                                  && UIDevice.current.userInterfaceIdiom != .phone // only iPhone
```

Can leave it as `.pad` but if you only want iPhone then I am thinking .phone might be better.